pepys_import/core/store/sqlite_db.py

```diff
@@ -78,25 +78,51 @@ class HostedBy(BaseSpatiaLite):
 
 
 class Sensor(BaseSpatiaLite):
-    __tablename__ = "Sensors"
+    __tablename__ = constants.SENSOR
     table_type = TableTypes.METADATA
     table_type_id = 2
 
     sensor_id = Column(Integer, primary_key=True)
     name = Column(String(150), nullable=False)
     sensor_type_id = Column(Integer, nullable=False)
-    platform_id = Column(Integer, nullable=False)
+    host = Column(Integer, nullable=False)
     created_date = Column(DateTime, default=datetime.utcnow)
 
+    @classmethod
```

Comment:
@BarisSari - `find_sensor` and it's equivalent in `postgres_db.py` file look the same. Shouldn't there be a single instance of the method?

Reply:
@IanMayo Since `get_sensor()` has to be a method of `Platform` class, I didn't want to put `find_sensor` to `data_store.py`. It could be confusing to have `find_sensor` method in data_store.py and `get_sensor` method in `postgres_db or sqlite_db`. Or I didn't want to implement it for only `sqlite or postgresdb` and call it from another because of the same reason. However, you're right that it's equivalent. If it doesn't seem right, we can find another solution.